test/parallel/test-process-cpuUsage.js

```diff
@@ -33,27 +33,57 @@ for (let i = 0; i < 10; i++) {
   assert(diffUsage.system >= 0);
 }
 
+const invalidUserArgument = new RegExp('^TypeError: value of user ' +
```

Comment:
@cjihrig,  honestIy would like to do it that way  ...  but can not find consensus about identation, got requests  to change things when doing something like that in previous commits ...  is there any document detailing the official code style? the linter seems to be fine with the previous commits but reviewers don't agree...  so... would really love to understand when is fine to use an identation like the one you are suggesting and when I should keep it in line after the =